source/Camera/InfoUtilOther/WarmupSample/WarmupSample.cs

```diff
@@ -0,0 +1,82 @@
+﻿
+/*
+This example shows a basic warm up method for a Zivid camera.
```

Comment:
https://github.com/zivid/zivid-csharp-samples/pull/42/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.md#r703618071

"This example shows a basic warm-up method for a Zivid camera with specified time and capture cycle"